### PR TITLE
Handle collectionFormat multi in parameters

### LINKED ISSUE #118: Validation fails when schema uses multi collection type for inquery parameters
When validating document described by schema having this fragment

``` json
 "parameters": [
      {
          "name": "id",
          "in": "query",
          "description": "Filter the results by id. Multiple ids may be supplied.",
          "type": "array",
          "items": { "type": "integer" },
          "collectionFormat": "multi"
      },
```

validator throws an exception:

```
python3.5/site-packages/flex/validation/common.py", line 407, in generate_value_processor
    delimeter = DELIMETERS[collectionFormat]
KeyError: 'multi'
```

swagger spec file is validated by swagger.io

### PR DESCRIPTION
**Pull request by do3cc**
multiple cats

Fixes #118

### CODE REVIEW

**Comment by do3cc** [comment]
WIP, don't merge

**Comment by do3cc** [comment]
Now not WIP any more. Ready for review.

File: flex/validation/common.py [hunk 1]
```diff
@@ -436,10 +437,13 @@ def generate_value_processor(type_, collectionFormat=None, items=None, **kwargs)
...
-            # split the string based on the delimeter specified by the
-            # `collectionFormat`
-            processors.append(operator.methodcaller('split', delimeter))
+            if collectionFormat in DELIMETERS:
+                delimeter = DELIMETERS[collectionFormat]
+                # split the string based on the delimeter specified by the
+                # `collectionFormat`
+                processors.append(operator.methodcaller('split', delimeter))
+            else:
+                assert collectionFormat == MULTI, "Unsupported collectionFormat"
```
Comment (pipermerriam) [inline flex/validation/common.py]: Can you change this to instead raise a `TypeError`?

Comment (do3cc) [inline flex/validation/common.py]: Done. I've also changed the wording to make it clear that the error might not be in the file to be checked.

**Comment by pipermerriam** [comment]
Available in `6.6.0`

**Comment by do3cc** [comment]
Thank you!

**Pull request merged**